dolphinscheduler-service/src/main/java/org/apache/dolphinscheduler/service/quartz/QuartzExecutors.java

```diff
@@ -70,9 +70,10 @@ public static QuartzExecutors getInstance() {
       synchronized (QuartzExecutors.class) {
         // when more than two threads run into the first null check same time, to avoid instanced more than one time, it needs to be checked again.
         if (INSTANCE == null) {
-          INSTANCE = new QuartzExecutors();
+          QuartzExecutors quartzExecutors = new QuartzExecutors();
           //finish QuartzExecutors init
-          INSTANCE.init();
+          quartzExecutors.init();
+          INSTANCE = quartzExecutors;
         }
```

Comment:
Fully initialize "INSTANCE" before assigning it.

https://sonarcloud.io/project/issues?id=apache-dolphinscheduler&issues=AW9_bPn7ek8Jf6Q3VCKX&open=AW9_bPn7ek8Jf6Q3VCKX